recipe/meta.yaml

```diff
@@ -50,3 +51,4 @@ about:
 extra:
     recipe-maintainers:
         - ocefpaf
+        - stuarteberg
```

Comment:
Per your [discussion]( https://github.com/conda-forge/staged-recipes/pull/374#discussion_r59987112 ) with @ocefpaf this will add you as a maintainer to the libtiff-feedstock, @stuarteberg. For an explanation of what that entails, see this [comment]( https://github.com/conda-forge/staged-recipes/pull/374#issuecomment-211051020 ).